inventory/host_vars/hetznercloudgw.freifunk-mwu.de

```diff
@@ -0,0 +1,31 @@
+---
+magic: 104
+ipv4_dhcp_range: 8
```

Comment:
Diese DHCP-Range ist schon durch Uffschnitt in Benutzung. Die Ranges 10-14 sind noch frei.